mappings/net/minecraft/world/GameRules.mapping

```diff
@@ -65,27 +65,30 @@ CLASS net/minecraft/class_1928 net/minecraft/world/GameRules
 		METHOD method_20776 setFromArgument (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)V
 			ARG 1 context
 			ARG 2 name
-		METHOD method_20777 setFromString (Ljava/lang/String;)V
+		METHOD method_20777 deserialize (Ljava/lang/String;)V
```

Comment:
Eh, `read` and `write` instead?

Reply:
`write` would be a poor name, as it is a no-args method that returns the serialized form. With the proposed name you would call `rule.serialize()`